src/sentry/static/sentry/app/views/settings/account/accountSecurity/components/recoveryCodes.jsx

```diff
@@ -47,19 +47,20 @@ class RecoveryCodes extends React.Component {
 
           <Actions>
             <Clipboard hideUnsupported value={formattedCodes}>
-              <Button size="small">
-                <InlineSvg src="icon-copy" />
+              <Button size="small" label="copy">
```

Comment:
We should translate these strings

```suggestion
              <Button size="small" label={t('copy'}>
```